src/iter/mod.rs

```diff
@@ -101,10 +107,10 @@ mod tests {
     fn values() {
         let map = HashMap::<usize, usize>::new();
 
-        let mut guard = epoch::pin();
+        let guard = map.guard();
         map.insert(1, 42, &guard);
         map.insert(2, 84, &guard);
-        guard.repin();
```

Comment:
Ah, interesting, there's no notion of re-pinning to allow previously-protected garbage to be collected in `seize`?

Reply:
No there isn't, because the reference counting is quite granular already.

Reply:
Hmm, but if all borrows are tied to a given `&Guard`, then without repin a reader has no way to signal that reclaiming the protected objects are okay except to drop the `Guard` and grab a new one. Which, efficiency aside, can be fairly annoying and require something like `std::mem::replace`.

Reply:
Hm, that is true. I'll see if I can add a `repin` operation.

Reply:
Added `flush` in ad9d5f29acce9a81b11f0e1875c39f45b6dae53e.